### PR TITLE
feat(toasts): Add toast molecule

### PR DESCRIPTION
**Pull request by gabrielhl**
Add a toast molecule with same variants as messages.

### CODE REVIEW

File: src/styles/modules/_toast.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transition: all linear .3s;
+    z-index: 9000;
+
+    &.dc-toast--hidden {
+        // For animating the toast entry/leave from the screen
+        top: -36px;
+    }
```
Comment (rbarilani) [inline src/styles/modules/_toast.scss]: What if the toast is more "higher"?

Comment (gabrielhl) [inline src/styles/modules/_toast.scss]: Good catch. I think a toast should have at max one line so I will force a max-height and set the correct overflow mode.

File: src/styles/modules/_toast.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.dc-toast {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transition: all linear .3s;
+    z-index: 9000;
+
```
Comment (rbarilani) [inline src/styles/modules/_toast.scss]: @gabrielhl use a variable for the z-index.

File: src/styles/modules/_toast.scss (hunk 1 above)
Comment (rbarilani) [inline src/styles/modules/_toast.scss]: @gabrielhl do you think a line is enough when on mobile screen?

**Comment by rbarilani** [comment]
@gabrielhl can we handle "responsiveness"?
@gabrielhl can we add variations for positioning?

.... check this out... http://codepen.io/hal9087/pen/Bjyxqj

**Comment by gabrielhl** [comment]
@rbarilani All good now, I believe we can merge this. We have the top/bottom positioning and it works if the text line wraps in case of small viewport.

**Pull request merged**